### PR TITLE
Sync Ruby Advisory DB

### PR DESCRIPTION
**Pull request by joshbuker**
Generated with: https://github.com/cloudsecurityalliance/gsd-tools/pull/200

### CODE REVIEW

File: 2007/6xxx/GSD-2007-6183.json [hunk 1]
```diff
@@ -202,11 +202,15 @@
                 "CVE_data_version": "4.0",
                 "nodes": [
                     {
-                        "children": [],
+                        "children": [
+
+                        ],
```
Comment (joshbuker) [inline 2007/6xxx/GSD-2007-6183.json:207]: Oh, I thought my script did the opposite of this...

Okay, I don't like this, and will see if I can get the JSON serializer to stop making useless newlines.

File: 2007/6xxx/GSD-2007-6183.json [hunk 2]
```diff
@@ -393,6 +435,54 @@
...
+                        "purl": "pkg:gem/gtk2"
+                    }
+                }
+            ],
+            "references": [
+                {
+                    "type": "WEB",
+                    "url": "https://nvd.nist.gov/vuln/detail/CVE-2007-6183"
+                }
+            ]
```
Comment (joshbuker) [inline 2007/6xxx/GSD-2007-6183.json:485]: Here is the meat and potatoes of the output.

File: 2009/4xxx/GSD-2009-4123.json [hunk 3]
```diff
@@ -53,6 +53,46 @@
...
+            "details": "A security problem involving peer certificate verification was found where\nfailed verification silently did nothing, making affected applications\nvulnerable to attackers. Attackers could lead a client application to believe\nthat a secure connection to a rogue SSL server is legitimate. Attackers could\nalso penetrate client-validated SSL server applications with a dummy\ncertificate.\n",
+            "affected": [
+                {
+                    "package": {
+                        "ecosystem": "RubyGems",
+                        "name": "jruby-openssl",
+                        "purl": "pkg:gem/jruby-openssl"
+                    }
+                }
+            ],
```
Comment (joshbuker) [inline 2009/4xxx/GSD-2009-4123.json:89]: Unfortunately I couldn't figure out how to sanely map the way ruby-advisory-db uses patched/unaffected into the introduced/fixed ranges OSV supports.

It's probably possible, and I'm just too sleep deprived atm to figure it out.

File: 2007/6xxx/GSD-2007-6183.json (hunk 1 above)
Comment (joshbuker) [reply 2007/6xxx/GSD-2007-6183.json]: Fixed.

**Comment by joshbuker** [comment]
@jasnow Good catch on the advisory count. I saw 477 and assumed it simply capped out - not realizing my counter was counting the wrong thing.

**Review by kurtseifried — APPROVED** [review]

**Pull request merged**